example/pubspec.yaml

```diff
@@ -56,7 +59,4 @@ flutter:
   #         weight: 700
   #
   # For details regarding fonts from package dependencies,
-  # see https://flutter.io/custom-fonts/#from-packages
-environment:
-  sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=1.10.0 <2.0.0" 
```

Comment:
I noticed this `flutter: ">=1.10.0 <2.0.0"` line was removed. Do we need to keep it? Or does `sdk: ">=2.12.0 <3.0.0"` cover this?

Reply:
Yeah, this is not needed, SDK version already cover it